config/routes.rb

```diff
@@ -6,6 +6,8 @@
   end
   root 'streams#index'
 
+  get '/stream_subscription' => 'streams#subscription'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.